data-prepper-plugins/common/src/main/java/org/opensearch/dataprepper/common/sink/DefaultSinkMetrics.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.common.sink;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.DistributionSummary;
+import org.opensearch.dataprepper.metrics.PluginMetrics;
+
+public class DefaultSinkMetrics implements SinkMetrics {
+    static final String DEFAULT_EVENT_NAME = "Event";
+    static final String SINK_REQUESTS_SUCCEEDED = "SinkRequestsSucceeded";
+    static final String SINK_REQUESTS_FAILED = "SinkRequestsFailed";
+    static final String SINK_REQUEST_LATENCY = "SinkRequestLatency";
+    static final String SINK_RETRIES = "SinkRetries";
+    static final String SINK_REQUEST_SIZE = "SinkRequestSize";
+    private final Counter sinkRequestsSucceeded;
+    private final Counter sinkRequestsFailed;
+    private final Counter sinkEventsSucceeded;
+    private final Counter sinkEventsFailed;
+    private final Counter sinkEventsDropped;
+    private final Counter sinkRetries;
+    private final DistributionSummary sinkRequestLatency;
+    private final DistributionSummary sinkRequestSize;
+    private final DistributionSummary sinkEventSize;
+
+    public DefaultSinkMetrics(final PluginMetrics pluginMetrics, final String sinkPrefix, final String eventName) {
+        this.sinkRequestsSucceeded = pluginMetrics.counter(sinkPrefix + SINK_REQUESTS_SUCCEEDED);
```

Comment:
Why do we need a `sinkPrefix`? The plugin name should be part of the name already.